galaxy.yml

```diff
@@ -0,0 +1,20 @@
+namespace: community
+name: FIXME
+version: 0.1.0
+readme: README.md
+authors:
+  - YOUR NAME (github.com/YOURGITHUB)
+description: null
+license_file: COPYING
+tags:
+  - acme
+  - certificate
+  - community
+repository: https://github.com/ansible-collections/community.REPO_NAME
+#documentation: https://github.com/ansible-collection-migration/community.REPO_NAME/tree/main/docs
+homepage: https://github.com/ansible-collections/community.REPO_NAME
+issues: https://github.com/ansible-collections/community.REPO_NAME/issues
+build_ignore:
+  - 'community-*-*.tar.gz'
+  - .gitignore
+  - changelogs/.plugin-cache.yaml
```

Comment:
I'm not really sure whether these are needed, or whether `.gitignore` with all its contents is automatically used. (So far I never checked this, and just kept these entries around, but for the template it's probably better to verify it first :) )

Reply:
Looks like https://github.com/ansible/ansible/commit/f8f76628500052ad3521fbec16c073ae7f99d287#diff-2de3c8cc3e7fae814d02b3058f0c8f9eR610 ignores the previously built tarballs.

Reply:
Doesn't appear that `.gitignore` is looked at when doing `ansible-galaxy collection build`, so need to keep some of these